tests/fakes.py

```diff
@@ -0,0 +1,17 @@
+class FakeRedisClient(object):
```

Comment:
I'm not so sure that we need this when we can just mock the Redis Wrapper itself. See https://github.com/alexk307/redis_cache/blob/master/tests/test_redis_cache.py#L18 for example